### ISSUE TITLE
Error when using this awesome module in Electron

### ISSUE THREAD

**Issue by aabuhijleh** [simonbuchan/native-reg#7]
Hi,

When I use this module in pure node, everything is fine.

However, when I load it  in an Electron app, the app crashes when hitting `reg.getValue` 

This is how I'm using it:

```ts
function getOutlookVersionPresent(): void {
  console.info("getOutlookVersionPresent - IN");
  try {
    const key = reg.openKey(
      reg.HKLM,
      "SOFTWARE\\Classes\\Outlook.Application\\CurVer",
      reg.Access.READ
    );
    console.warn("Something goes wrong here (In Electron)");
    const version = reg.getValue(key, "", ""); // default value
    console.info(`getOutlookVersionPresent - version:[${version}]`);
    reg.closeKey(key);
  } catch (e) {
    console.error("getOutlookVersionPresent - failure", e);
  }
}
```

The app crashes and this error message is printed:

```sh
#
# Fatal error in , line 0
# ignored
#
#
#
#FailureMessage Object: 00F7E16Cnpm ERR! code ELIFECYCLE
```

This is my [repository](https://github.com/aabuhijleh/native-reg-electron-issue).

To reproduce this issue, you just have to fire these commands:

```sh
$ npm install
$ npm start
```

**Electron version (Windows ofc)**: 8.0.3 (tried both x86 & x64)
**native-reg version**: 0.3.3

I also tried to not use prebuilds and build the package from source but that didn't work either.

In pure node, this works perfectly.

**What am I doing wrong?**

Thanks in advance!

**Comment by simonbuchan** [comment]
Thanks for the report!

TL;DR: Use `reg.queryValue(key, "")` instead, this seems to be specific to using `getValue()` to get the default value.

This is a bit confusing: I can reproduce it, but in both electron and node.
When running in node, I get a stack complaining about some internal validation (related to WASM!?), so I'm currently suspecting that the V8 runtime in electron is newer than the node version you are testing with, and it's not electron specific. I'll look into it.

**Comment by simonbuchan** [comment]
Should be fixed in new 0.3.4

**Comment by aabuhijleh** [comment]
> TL;DR: Use `reg.queryValue(key, "")` instead, this seems to be specific to using `getValue()` to get the default value.
> 
> This is a bit confusing: I can reproduce it, but in both electron and node.
> When running in node, I get a stack complaining about some internal validation (related to WASM!?), so I'm currently suspecting that the V8 runtime in electron is newer than the node version you are testing with, and it's not electron specific. I'll look into it.

You're right I'm using `node -v 12.0.5` where Electron 8 uses `node -v 12.13.0`.

`reg.queryValue` works perfectly. Thanks.

**Comment by simonbuchan** [comment]
Have you tried with the new publish, too?

**Comment by aabuhijleh** [comment]
Yes and this issue is resolved. Thanks.
